spec/itemMockCtrl.spec.js

```diff
@@ -0,0 +1,154 @@
+const chai = require('chai');
+const expect = chai.expect;
+const assert = chai.assert;
+const sinon = require('sinon');
+const mocha = require('mocha');
+const describe = mocha.describe;
+const beforeEach = mocha.beforeEach;
+const afterEach = mocha.afterEach;
+const it = mocha.it;
+const ItemMockCtrl = require('../server/controllers/ItemMockCtrl');
+const R = require('ramda');
+describe('itemMockCtrl', () => {
+  describe('getById', () => {
+    it('should return undefined if the items array is empty', () => {
+      const items = [];
+      const id = 1;
+      const result = ItemMockCtrl.getById(id, items);
+      expect(result).to.equal(undefined);
+    });
+    it('should return undefined if the ID does not exist ', () => {
+      const items = [
+        { id: 1, name: 'test', category: 'A', group: 'group1' },
+        { id: 3, name: 'test3', category: 'C', group: 'group3' }
+      ];
+      const id = 2;
+      const result = ItemMockCtrl.getById(id, items);
+      expect(result).to.equal(undefined);
+    });
+    it('should return the correct item if exists', () => {
+      const expectedItem = { id: 1, name: 'test', category: 'A', group: 'group1' };
+      const items = [
+        { id: 1, name: 'test', category: 'A', group: 'group1' },
+        { id: 3, name: 'test3', category: 'C', group: 'group3' }
+      ];
+      const id = 1;
+      const findStub = sinon.stub(R, 'find').returns(expectedItem);
+      const result = ItemMockCtrl.getById(id, items);
+      expect(result).to.deep.equal(expectedItem);
+      findStub.restore();
```

Comment:
Il n'y a pas besoin de passer par une librairie tierce, tu peux simplement faire ça : 
```suggestion
      const expectedItem = { id: 1, name: 'test', category: 'A', group: 'group1' };

      ItemMockCtrl.create(expectedItem);
      
      const result = ItemMockCtrl.getById(id, items);
      expect(result).to.deep.equal(expectedItem);
```